### PR TITLE
osdocs-634 and osdocs-639 deleting gcp and azure clusters

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://jira.coreos.com/browse/OSDOCS-634
https://jira.coreos.com/browse/OSDOCS-639

@abhinavdahiya  says that there's no difference in procedure between uninstalling for AWS, Azure, and GCP IPI. I had a chat with @aheslin about the structure for the uninstalling processes and made a generic module and an assembly for each cloud platform.

@e-tienne, @wmengRH, as you test out Azure and GCP, let me know if you see anything different in the cluster deletion process or if these methods are fine.

### CODE REVIEW

**Comment by wmengRH** [comment]
LGTM for GCP part.

**Comment by e-tienne** [comment]
Hello,

For **IPI-on-Azure**, I confirm that the `openshift-install destroy cluster` procedure is standard, but I would remove the following section:

> If not all of the cluster resources are removed from your cloud, you can use this UUID
> to locate them and remove them.

and replace it with the procedure to rebuild a lost  `metadata.json` in order to run the `openshift-install destroy cluster` command. Example: https://access.redhat.com/solutions/3826921

For the other versions of the IPI, this might need to be tested and verified. I've been told that older versions of 4.x can use a different `metadata.json`.

I hope this helps.

**Comment by kalexand-rh** [comment]
@openshift/team-documentation PTAL

**Review by bobfuru — COMMENTED** [review]
LGTM!

File: modules/installation-uninstall-clouds.adoc [hunk 1]
```diff
@@ -1,28 +1,23 @@
...
 
-You can remove a cluster that you installed on Amazon Web Services (AWS).
+You can remove a cluster that uses installer-provisioned infrastructure from
+your cloud.
 
 .Prerequisites
 
 * Have a copy of the installation program that you used to deploy the cluster.
+* Have the files that the installation program generated when you created your
+cluster.
```
Comment (mburke5678) [inline modules/installation-uninstall-clouds.adoc]: @kalexand-rh Will the user know what/where these files are?

Comment (kalexand-rh) [reply modules/installation-uninstall-clouds.adoc]: @mburke, they should. I have a note in the installation files that says "You must not delete the installation program or the files that the installation program creates. Both are required to delete the cluster."

**Comment by vikram-redhat** [comment]
@wmengRH all good in this PR for GCP?
@e-tienne ok from Azure perspective?

**Comment by wmengRH** [comment]
Yes, it is good for GCP part.
Thanks.

**Comment by e-tienne** [comment]
LGTM

**Pull request merged**